src/community/china/_os-settings.md

```diff
@@ -0,0 +1,106 @@
+{% assign id =  include.os | downcase -%}
+
+{% if id == 'windows' -%}
+{% assign shell = 'Powershell' -%}
+{% assign prompt = 'C:\>' -%}
+{% assign envvarset = 'C:\> $env:' -%}
+{% assign setpath = 'C:\> $env:PATH = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"' -%}
+{% assign comtoset = '$env:' -%}
+{% assign unzip='Extract-Archive' %}
+{% capture permaddexample -%}
+$newPath = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"
+[System.Environment]::SetEnvironmentVariable('Path',$newPath,User)
+[System.Environment]::SetEnvironmentVariable('PUB_HOSTED_URL','https://pub.flutter-io.cn',User)
+[System.Environment]::SetEnvironmentVariable('FLUTTER_STORAGE_BASE_URL','https://storage.flutter-io.cn',User)
+{% endcapture -%}
+{% else -%}
+{% assign shell = 'your terminal' -%}
+{% assign prompt = '$' -%}
+{% assign envvarset = '$ export ' -%}
+{% assign setpath = '$ export PATH="$PWD/flutter/bin:$PATH"' -%}
+{% assign comtoset = 'export' -%}
+{% assign unzip='unzip' %}
+{% capture permaddexample -%}
+cat <<EOT >> ~/.zprofile
+{{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+{{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+{{setpath}}
+EOT
+{% endcapture -%}
+{% endif -%}
+
+<div id="{{id}}" class="tab-pane
+  {%- if id == 'windows' %} active {% endif %}"
+  role="tabpanel" aria-labelledby="{{id}}-tab" markdown="1">
+
+1. Open a new Terminal window to prepare running scripts.
+
+1. Set `PUB_HOSTED_URL` to your mirror site.
+   In {{shell}}, run the following command:
+
+   ```terminal
+   {{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+   ```
+
+1. Set `FLUTTER_STORAGE_BASE_URL` to your mirror site.
+   In {{shell}}, run the following command:
+
+   ```terminal
+   {{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+   ```
+
+1. Download the Flutter archive from your mirror site.
```

Comment:
"Archive" implies the entire archive, perhaps "build" or "installation bundle".

```suggestion
1. Download your the Flutter installation bundle for
   your platform from your mirror site.
```

Reply:
A zip file or a tar file are archive file formats. Going to keep as-is.

Reply:
That's technically true, but it's quite confusing since it uses "archive" to mean two different things in this same step. It reads as if it's telling me to download the entire archive page.